Files/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# ==================================================================================================================
+#
+# Docker to ubuntu 18.04 base image (for required libraries) used by GCBM
+#
+# Building this Docker:
+#   docker build --build-arg NUM_CPU=14 -t moja/gcbm:ubuntu-18.04 .
+#
+# ==================================================================================================================
+
+FROM moja/baseimage:ubuntu-18.04
```

Comment:
Is this supposed to be `mojaglobal/baseimage` instead?

I'm getting an access denied/does not exist error when I try to build it.
```
Sending build context to Docker daemon  4.096kB
Step 1/25 : FROM moja/baseimage:ubuntu-18.04
pull access denied for moja/baseimage, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

Reply:
I'm actually unable to build it with even this. 
8GB RAM is not enough. Ubuntu just totally freeezes up in the middle of the cmake build.

Reply:
I built the base image locally first, but yes - I think you should be able to switch to mojaglobal /baseimage to pull from Docker Hub.

Can you try moving the Dockerfile into a separate folder? I noticed it uses the local directory a the 'build context' which might be eating memory.